packages/core/lib/commands/develop/run.js

```diff
@@ -9,10 +9,10 @@ const runConsole = async (config, ganacheOptions) => {
   const Console = require("../../console");
   const { Environment } = require("@truffle/environment");
 
-  const commands = require("../index");
+  const commands = require("../commands");
   const excluded = new Set(["console", "develop", "unbox", "init"]);
```

Comment:
Thoughts on adding `db` to the exclusion list? Currently, it allows for `db serve` which will lock up the develop console and is a terrible user experience.

Reply:
I have no problem with that as long as we are sure we don't want it running in this context. The real question is 'do we want to be able to run db in the repl?'.

Reply:
I vote `no` because it stops REPL behavior, and currently we can't stop the process with ctrl-c, ctrl-z, or other keystrokes within the locked-up terminal.

Reply:
Ok, I'm going to blacklist this here unless someone else wants to voice a dissenting opinion here.